OgreMain/src/OgreAnimationTrack.cpp

```diff
@@ -196,13 +196,10 @@ namespace Ogre {
         // Pre-allocate memory
         mKeyFrameIndexMap.resize(keyFrameTimes.size());
 
-        size_t i = 0, j = 0;
```

Comment:
I think the previous version of this was more readable..

Reply:
reverted